tomviz/HistogramWidget.h

```diff
@@ -76,6 +77,8 @@ public slots:
   vtkWeakPointer<vtkTable> m_inputData;
 
   QVTKGLWidget* m_qvtk;
+
+  QWidget* m_presetDialog = nullptr;
```

Comment:
This might should be a `PresetDialog*` instead of a `QWidget*`, so that you'd be able to call derived class functions if any get added.